dod-advana/gamechanger-web
# Task/mainpage portfolio

## Description

- split out ViewHeader into modules. Includes sequelize and sql files for this change
- change how jbook reviews are saved and retrieved in PG
- handle multiple reviews on profile page and main view
- make portfolios selectable on profile page and main view
- (on main page, assume reviews are in an object coming from ES)

## Related Issue/Ticket

https://jira.di2e.net/browse/UOT-144654
https://jira.di2e.net/browse/UOT-144656

## Testing instructions

- navigate to /jbook, see that you can use the portfolio selector (shouldn't change anything atm)
- click the Open button on a card, see that you can use the portfolio selector
- go to the first form on the page and save a review. if you change the portfolio, it should change the information displayed n the form and you should be able to save to each portfolio individually
